main.py

```diff
@@ -88,6 +90,8 @@ def main():
         description_dict=description_dict,
         decontamination_ngrams_path=args.decontamination_ngrams_path,
         check_integrity=args.check_integrity,
+        accelerate=args.accelerate,
```

Comment:
Maybe `use_accelerate` is a better name for this arg?

Reply:
Agrred!

Reply:
SHould be fixed in 30b6b1bd54b0f406507cc1cfac69e76f4a0551f2